### ISSUE TITLE
The `abstract-decoratee` check must be warning but not error

### ISSUE THREAD

**Issue by maxonfjvipon** [objectionary/lints#28]
Consider the code which is get after `unphi` operation:
```
# This is the default 64+ symbols comment in front of named abstract object.
[] > a
  # This is the default 64+ symbols comment in front of named abstract object.
  [] > @
    # This is the default 64+ symbols comment in front of named abstract object.
    [] > @
      # This is the default 64+ symbols comment in front of named abstract object.
      [] > @
        for. > @
          malloc.
            eolang.
              org.
                Q
          number.:0
            eolang.
              org.
                Q
            00-00-00-00-00-00-00-00:0
```

It's valid and it works but it fails with error: Abstract object can't be used as a decoratee.
I believe this check must be a warning so it's kind of bad practice to write a code in a such way
@yegor256 WDYT?

**Comment by maxonfjvipon** [comment]
@yegor256 I'm not sure since it breaks structure of original EO code which is for example a unit test with such design

**Comment by maxonfjvipon** [comment]
@yegor256 yes, it helps, but still I think `abstract-decoratee` is not an error but rather bad way to write EO code. The same as sparse decoration

**Comment by yegor256** [comment]
@maxonfjvipon OK, let's change its severity to `warning`

**Comment by h1alexbel** [comment]
@yegor256 can I take this one?
